README.md

```diff
@@ -10,6 +10,12 @@
 [![npm Package Version](https://badge.fury.io/js/%40ui5%2Fcli.svg)](https://www.npmjs.com/package/@ui5/cli)
 [![Coverage Status](https://coveralls.io/repos/github/SAP/ui5-cli/badge.svg)](https://coveralls.io/github/SAP/ui5-cli)
 
+## Deprecation
```

Comment:
Why is this text different from the other modules? e.g. https://github.com/SAP/ui5-fs/pull/506/files

Reply:
My point was that this, along with the ui5-tooling repo those are the "visible" repos for the outer world.
I checked some deprecated packages on NPM and they breifly follow this pattern- a link to a deprecated section in README.md. Take for example, this one: https://www.npmjs.com/package/source-map-url

As far as I got it, we'd put a deprecated message on this package: https://www.npmjs.com/package/@ui5/cli/v/latest-2
so, I wanted to have this link to the deprecated section of the README file.